src/components/pages/SignUp.vue

```diff
@@ -1,16 +1,217 @@
 <template>
-  <div class="">
-    <h1>Sign Up Page</h1>
+  <div class="sign-up">
```

Comment:
Components này anh tách nhỏ xuống, hiện tại đang quá to, rất khó maintain, tầng organism mình còn chưa dùng đến